components/country/overview.js

```diff
@@ -0,0 +1,171 @@
+    <React.Fragment>
+      <SectionHeader>
+        <SectionHeader.Title name='overview'>
+          <FormattedMessage id='Country.Heading.Overview' />
+        </SectionHeader.Title>
+      </SectionHeader>
+      <BoxWithTitle title={<FormattedMessage id='Country.Overview.Heading.NwInterference' />}>
+        <Flex flexWrap='wrap'>
+          <NwInterferenceStatus width={[1, 1/2]} color={middleboxCount && 'violet8'}>
+            <NettestGroupMiddleBoxes size={32} />
+            <FormattedMessage
+              id={ getStatus(middleboxCount, 'Country.Overview.NwInterference.Middleboxes') }
+              values={{ middleboxCount }}
+            />
+          </NwInterferenceStatus>
+          {/* <NwInterferenceStatus width={[1, 1/2]} color={circumventionTools && 'pink6'}>
+            <FormattedMessage
+              id={ getStatus(circumventionTools, 'Country.Overview.NwInterference.CircumventionTools') }
+              values={{ circumventionTools }}
+            />
+          </NwInterferenceStatus> */}
+          <NwInterferenceStatus width={[1, 1/2]} color={imCount && 'yellow9'}>
+            <NettestGroupInstantMessaging size={32} />
+            <FormattedMessage
+              id={ getStatus(imCount, 'Country.Overview.NwInterference.IM') }
+              values={{ imCount }}
+            />
+          </NwInterferenceStatus>
+          <NwInterferenceStatus width={[1, 1/2]} color={blockedWebsitesCount && 'indigo5'}>
+            <NettestGroupWebsites size={32} />
+            <FormattedMessage
+              id={ getStatus(blockedWebsitesCount, 'Country.Overview.NwInterference.Websites') }
+              values={{ blockedWebsitesCount }}
+            />
+          </NwInterferenceStatus>
+        </Flex>
+      </BoxWithTitle>
+      <Heading h={4}>
+        <FormattedMessage id='Country.Overview.Heading.TestsByClass' />
+      </Heading>
+      <TestsByGroup
+        fetchTestCoverageData={fetchTestCoverageData}
+        testCoverage={testCoverage}
+        networkCoverage={networkCoverage}
+      />
+      <BoxWithTitle title={<FormattedMessage id='Country.Overview.FeaturedResearch' />}>
+        {
+          (featuredArticles.length === 0)
+            ? <FormattedMessage id='Country.Overview.FeaturedResearch.None' />
+            : <ul>
+              <li>
+                <FeaturedArticle
+                  link='https://ooni.io/post/cuba-referendum/'
+                  title='Cuba blocks independent media amid 2019 constitutional referendum'
+                />
+              </li>
+              <li>
+                <FeaturedArticle
+                  link='https://ooni.io/post/ooni-software-development-guidelines/'
+                  title='OONI Software Development Guidelines'
+                />
+              </li>
+              <li>
+                <FeaturedArticle
+                  link='https://ooni.io/post/ooni-probe-android-200-incident/'
+                  title='OONI Probe Android 2.0.0 Incident Report'
```

Comment:
We probably don't want to be showing this as featured research :P.

Jokes aside we should file a ticket related to doing some work on extracting posts that are relevant to a particular country.